sections/semantics-sections.include

```diff
@@ -449,6 +460,12 @@
 
   The <{nav}> element <a>represents</a> a section of a page that links to other pages or to
   parts within the page: a section with navigation links.
+  
+   <p>The semantics of the <{nav}> element may be conveyed to users of Assistive Technology. 
```

Comment:
We prefer active voice:

> Assistive Technology may convey the semantics of the <{nav}> element to users.

…

Not critical for now, but in general we should try to get this consistent.